src/screens/OrganizationEvents/OrganizationEvents.tsx

```diff
@@ -135,74 +127,25 @@ function OrganizationEvents(): JSX.Element {
 
   /* istanbul ignore next */
 
-  const handleSearchByTitle = (e: any) => {
-    const { value } = e.target;
-    const filterData = {
-      organization_id: currentUrl,
-      title_contains: value,
-    };
-    refetch(filterData);
-  };
-  const handleSearchByDescription = (e: any) => {
-    const { value } = e.target;
-    const filterData = {
-      organization_id: currentUrl,
-      description_contains: value,
-    };
-    refetch(filterData);
-  };
-  const handleSearchByLocation = (e: any) => {
-    const { value } = e.target;
-    const filterData = {
-      organization_id: currentUrl,
-      location_contains: value,
-    };
-    refetch(filterData);
-  };
-
-  const debouncedHandleSearchByTitle = debounce(handleSearchByTitle);
-  const debouncedHandleSearchByDescription = debounce(
-    handleSearchByDescription
-  );
-  const debouncedHandleSearchByLocation = debounce(handleSearchByLocation);
+  // const handleSearchByTitle = (e: any) => {
+  //   const { value } = e.target;
+  //   const filterData = {
+  //     organization_id: currentUrl,
+  //     title_contains: value,
+  //   };
+  //   refetch(filterData);
+  // };
 
+  // const debouncedHandleSearchByTitle = debounce(handleSearchByTitle);
```

Comment:
Remove the commented code

Reply:
This is search functionality for event which is already implemented, we may require it in future. Hence I commented it for now.